src/tree/RevisionItem.ts

```diff
@@ -44,7 +44,7 @@ export class RevisionItem implements RevisionsItemModel {
             label: this.revision.name,
             iconPath: this.iconPath,
             description,
-            contextValue: 'revision',
+            contextValue: `${this.revision.active ? 'active' : 'inactive'};revision`,
```

Comment:
The context value string should be created using the `createContextValue` util. It makes sure the values are sorted and deduped.